tools/lint-all

```diff
@@ -117,8 +117,12 @@ def check_pyflakes():
 
 def custom_check_file(fn, rules, skip_rules=[]):
     failed = False
-    for i, line in enumerate(open(fn)):
+    flag = False
+    f=open(fn,'r+')
```

Comment:
`flag` should be a more clear variable name.

Also, why did you change way the file is opened?  Seems like you can remove that change.

Finally, I'm not sure you need the lastLine variable, and I think it needs to set `failed` to actually cause a failure.